### DISCUSSION TITLE
What is the official/expected mobile solution for frigate?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#3271] [Ask A Question] [answered]
Loading the web UI doesn't work on Android or iOS (should it?). I tried the unofficial android app but playback was black. Is there a mobile solution for frigate?

**Comment by NickM-27** [comment, marked as answer]
Yeah, the web ui works just fine for mobile. Does none of it work in the browser for you? What browser are you running? Do you see any errors?

**Reply by thinkloop** [reply]
I'm not sure what's going on but it works today, in both firefox and chrome for android. It does *not* work on safari on my old iPad which is stuck on iOS 12.5.5.

**Reply by NickM-27** [reply]
Huh that is odd, the iPad not working makes sense as 12.5.5 is rather old and it is possible that safari does not support some newish javascript features that frigate relies on. Not sure if you can get an error from the iPad safari but that could be helpful. 

Does chrome on that iPad work (since you specifically mentioned safari)?
